src/settings/mod.rs

```diff
@@ -328,4 +346,26 @@ impl Settings {
             Err(error) => Err(error),
         }
     }
+
+    /// Create rocket configuration based on the `NODE_ENV` environment
+    /// variable. Defaults to `dev` mode if `NODE_ENV` is not set.
+    pub fn build_rocket_config(&self) -> Result<RocketConfig, RocketConfigError> {
+        match env::var("NODE_ENV").as_ref().map(String::as_ref) {
+            Ok("production") => RocketConfig::build(RocketEnvironment::Production)
+                .address(self.host.0.clone())
+                .port(self.port.clone())
+                .log_level(LoggingLevel::Off)
+                .finalize(),
+            Ok("staging") => RocketConfig::build(RocketEnvironment::Staging)
+                .address(self.host.0.clone())
+                .port(self.port.clone())
+                .log_level(LoggingLevel::Critical)
+                .finalize(),
+            Ok("dev") | _ => RocketConfig::build(RocketEnvironment::Development)
+                .address(self.host.0.clone())
+                .port(self.port.clone())
+                .log_level(LoggingLevel::Normal)
```

Comment:
There is a `Debug` level here, but I thought that was a bit too much. This is the default one and the one we were using previously, but just FYI.